packages/core/src/agent/didcomm/versions/v2/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { DecryptedMessageContext, EncryptedMessage, SignedMessage, MessageType } from '../../types'
+import type { AgentContext } from './../../../context/AgentContext'
+import type { DIDCommV2Message } from './DIDCommV2Message'
+
+export { DIDCommV2Message } from './DIDCommV2Message'
+export { DIDCommV2BaseMessage, DIDComV2BaseMessageConstructor } from './DIDCommV2BaseMessage'
+
+export interface PackMessageParams {
+  toDID?: string
+  fromDID?: string
```

Comment:
The from and to did are also defined in the DIDCommV2Mesasge right? Why do we need to redeclare them here?